src/blocks/mrc_opmode_details.ts

```diff
@@ -80,6 +86,24 @@ const OPMODE_DETAILS = {
     this.getField('NAME')?.setTooltip(Blockly.Msg.OPMODE_NAME_TOOLTIP);
     this.getField('GROUP')?.setTooltip(Blockly.Msg.OPMODE_GROUP_TOOLTIP);
   },
+  checkOpMode(this: OpmodeDetailsBlock, editor: Editor): void {
+    if (editor.isStepsInWorkspace() ||
+        editor.getMethodNamesAlreadyOverriddenInWorkspace().includes(PERIODIC_METHOD_NAME)) {
+      // Remove any previous warning.
+      this.setWarningText(null, WARNING_ID_STEPS_OR_PERIODIC_REQUIRED);
```

Comment:
This should be Blockly.Msg.WARNING_OPMODE_STEPS_OR_PERIODIC_REQUIRED

Reply:
This is the blockly warning id, not the message.

Reply:
Then why is this different than line 98?

Reply:
Oh, is the null removing the warning?